mParticle-BranchMetrics.podspec

```diff
@@ -13,8 +13,8 @@ Pod::Spec.new do |s|
     s.source           = { :git => "https://github.com/mparticle-integrations/mparticle-apple-integration-branchmetrics.git", :tag => s.version.to_s }
     s.social_media_url = "https://twitter.com/mparticle"
 
-    s.ios.deployment_target = "8.0"
+    s.ios.deployment_target = "9.0"
     s.ios.source_files      = 'mParticle-BranchMetrics/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 7.0'
-    s.ios.dependency 'Branch', '~> 0.33.0'
+    s.ios.dependency 'Branch', '~> 0.34.0'
```

Comment:
let's just open this up to the latest

Reply:
0.34 is the latest. I think we discussed this but I want to clarify